platform/ui/src/components/Button/Button.tsx

```diff
@@ -66,19 +67,20 @@ const Button = ({
   name,
   className,
   onClick,
+  tooltip = null,
```

Comment:
it's not button tooltip right? it is icon tooltip

Reply:
Fixed